scripts/aws/setupdb.sh

```diff
@@ -18,6 +18,8 @@ psql -c "CREATE EXTENSION IF NOT EXISTS postgis;"
 envdir /etc/nyc-trees.d/env /opt/app/manage.py migrate
 # Load block face data
 envdir /etc/nyc-trees.d/env /opt/app/manage.py loaddata $DIR/../src/nyc_trees/apps/survey/fixtures/blockface.json
+# Setup soft launch flag
+envdir /etc/nyc-trees.d/env /opt/app/manage.py loaddata $DIR/../src/nyc_trees/nyc_trees/fixtures/soft-launch-flag.json
```

Comment:
It would be nice to use the [Waffle CLI](http://waffle.readthedocs.org/en/latest/usage/cli.html) instead of Django fixtures.